app/src/main/java/com/jerboa/db/AppDB.kt

```diff
@@ -322,6 +327,9 @@ val MIGRATION_10_11 = object : Migration(10, 11) {
         database.execSQL(
             "ALTER TABLE AppSettings add column show_comment_action_bar_by_default INTEGER NOT NULL default 1",
         )
+        database.execSQL(
+            "ALTER TABLE AppSettings add column show_voting_arrows_in_list_view INTEGER NOT NULL default 1",
```

Comment:
@twizmwazin I'm ambivalent, but do you think these really needs a setting? I'm leaning towards no, that it should just always show it.

If it does, then it needs a separate migration. The only reason to mess with old migrations, is if something in them was broken AND we haven't done a release.

Reply:
I personally want a setting to disable it, if it wasn't added in this PR I would have done it myself soon. Boost also has a setting to control this.

I was under the impression for migrations that we would use a single migration between each released version. Otherwise, every single checkbox we add to the settings is going to result in a very long list of migrations very quickly. We haven't discussed this explicitly though, perhaps we should make a contributing guide that clearly establishes these guidelines.

Reply:
mmmk we can leave.

I don't mind tons of migrations, we're going to end up with a lot no matter what. The general rule is that a migration should be attached to a feature.

Its also **much** safer to have new migrations, than to edit previously committed ones. I've run into this in the past, where people carelessly edited existing deployed migrations, and broke production. 

If we need to later, we can organize them into their own file(s) if it gets out of hand. Diesel for rust does this automatically, and I'm surprised Room doesn't encourage you to do this already.

Reply:
@a1studmuffin So in that case break this out into its own migration if you would.